framework-plugins/lisk-framework-forger-plugin/test/integration/setup.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+require('../_setup');
+
+jest.setTimeout(30000);
```

Comment:
Is this value not too high for integration tests? Integration tests supposed to run some isolated part of functionality and assert.